content/publishing-yaml/distribution.md

```diff
@@ -68,11 +68,17 @@ publishing:
 
 Codemagic enables you to automatically publish your app to the `internal`, `alpha`, `beta` and `production` tracks on Google Play. In order to do so, you will need to set up a service account in Google Play Console and add the `JSON` key file to your Codemagic configuration file, see how to [set up a service account](../knowledge-base/google-play-api/).
 
+In case your application supports [in-app updates](https://developer.android.com/guide/playcore/in-app-updates) Codemagic allows you to set the update priority. In case in-app updates are not supported, `in_app_update_priority` can be left out of the codemagic.yaml, or be set to `0`.
+
+In addition, Codemagic supports [staged releases](https://support.google.com/googleplay/android-developer/answer/6346149?hl=en), allowing users to choose which fraction of the testers or users get access to the application. If an user wants to release a full version, `rollout_fraction` should be left out of the codemagic.yaml. 
```

Comment:
Instead of "full version", maybe it would be better to have "release to everyone"?
Eg. "To release to everyone, omit `rollout_fraction` from codemagic.yaml."